src/api_server/src/attest/amber.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2023 by Intel.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::Attest;
+use anyhow::*;
+use as_types::AttestationResults;
+use async_trait::async_trait;
+use jsonwebtoken::{decode, decode_header, jwk, DecodingKey, Validation};
+use kbs_types::{Attestation, Tee};
+use reqwest::header::{ACCEPT, CONTENT_TYPE};
+use serde::{Deserialize, Serialize};
+use std::fs::File;
+use std::io::BufReader;
+
+#[derive(Deserialize, Debug)]
+struct TdxEvidence {
```

Comment:
This is a structure that's now shared between 3 separate crates: `attestation-service`, `kbs`and `attestation-agent`. It is generated by the AA crate, so we should export the AA attester structures through an AA `kbc-types` crate or maybe use the `virtee/kbs-types`crate for it?